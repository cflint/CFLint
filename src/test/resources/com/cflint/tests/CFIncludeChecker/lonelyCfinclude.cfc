<cfcomponent>
 <cffunction>
  <cfinclude template="functions.cfm">
 </cffunction>
</cfcomponent>