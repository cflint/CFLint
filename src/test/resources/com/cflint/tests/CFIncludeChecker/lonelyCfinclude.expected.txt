{
  "version" : "",
  "timestamp" : 1526095021,
  "issues" : [ {
    "severity" : "WARNING",
    "id" : "AVOID_USING_CFINCLUDE_TAG",
    "message" : "AVOID_USING_CFINCLUDE_TAG",
    "category" : "CFLINT",
    "abbrev" : "AU",
    "locations" : [ {
      "file" : "src\\test\\resources\\com\\cflint\\tests\\CFIncludeChecker\\lonelyCfinclude.cfc",
      "fileName" : "lonelyCfinclude.cfc",
      "function" : "",
      "offset" : 30,
      "column" : 3,
      "line" : 3,
      "message" : "Avoid using <cfinclude> tags. Use components instead.",
      "variable" : "cfinclude",
      "expression" : "<cfinclude template=\"functions.cfm\">"
    } ]
  } ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ {
      "code" : "AVOID_USING_CFINCLUDE_TAG",
      "count" : 1
    } ],
    "countBySeverity" : [ {
      "severity" : "WARNING",
      "count" : 1
    } ]
  }
}